main.py

```diff
@@ -156,62 +156,66 @@ async def get_settings(ctx: discord.ApplicationContext):
-    try: c.execute("SELECT * FROM data WHERE guild_id = ?", (str(message.guild.id),))
-    except: return
-    data2 = c.fetchone()
-    if data is None: return
-    channel = message.guild.get_channel(int(data2[1]))
-    is_enabled = data2[2]
-    moderator_role_id = data2[3]
-    content = message.content
+  unbanned_words = ["kick", "kicks", "kicking"]
+  for word in unbanned_words:
+        if word in message.content:
+            return
+        if message.author == bot.user: return #if the message is sent by the bot, we don't want to moderate it, we don't want to moderate the bot,
+        try: c.execute("SELECT * FROM moderation WHERE guild_id = ?", (str(message.guild.id),))
+        except: return
+        data = c.fetchone()
+        try: c.execute("SELECT * FROM data WHERE guild_id = ?", (str(message.guild.id),))
+        except: return
+        data2 = c.fetchone()
+        if data is None: return
+        channel = message.guild.get_channel(int(data2[1]))
+        is_enabled = data2[2]
+        moderator_role_id = data2[3]
+        content = message.content
     #if the content starts with everything excpet something containing a space and then ! , we don't want to moderate it because it is a command
     #first we check if the content starts with everything and then !
     #with regex, we check if the content starts with anything and then !
     #we also do that with the manage_messages permission, so the moderators can't be moderated
-    if not content.startswith("MOD TEST"):
-        if message.author.guild_permissions.manage_messages: return #if the user is a moderator, we don't want to moderate him because he is allowed to say whatever he wants because he is just like a dictator
-        if message.author.guild_permissions.administrator: return #if the user is an administrator, we don't want to moderate him because he is allowed to say whatever he wants because he is a DICTATOR
-    else:
-        content = content.replace("MOD TEST ", "")
-        content = content.replace("MOD TEST", "")
-    if re.match(r".*!", content):
+        if not content.startswith("MOD TEST"):
+            if message.author.guild_permissions.manage_messages: return #if the user is a moderator, we don't want to moderate him because he is allowed to say whatever he wants because he is just like a dictator
+            if message.author.guild_permissions.administrator: return #if the user is an administrator, we don't want to moderate him because he is allowed to say whatever he wants because he is a DICTATOR
+        else:
+            content = content.replace("MOD TEST ", "")
+            content = content.replace("MOD TEST", "")
+        if re.match(r".*!", content):
     #if the match dosent contain a space, we don't want to moderate it because it is a command
-        if not " " in re.match(r".*!", content).group(0):
+            if not " " in re.match(r".*!", content).group(0):
+                return
+        if not is_enabled: return
```

Comment:
These are also unrelated to whitelist, more exceptions basically that doesn't need to be run for every word in the whitelist